engine/src/cmd/collide2/CSopcodecollider.cpp

```diff
@@ -27,12 +27,15 @@
 /*
  * Copyright (C) 2020 pyramid3d
  * Copyright (C) 2020-2022 Stephen G. Tuggy
+ * Copyright (C) 2025 Benjamen R. Meyer
  */
 
 
+#include "Opcode.h"
 #include "CSopcodecollider.h"
 #include "opcodeqsqrt.h"
 #include "opcodeqint.h"
+#include "vs_vector.h"
```

Comment:
I actually just got rid of vs_vector a few days ago in one of my PRs. vs_vector was a mistake from the beginning. It inherits from std::vector, which is not intended to be inherited from. (std::vector doesn't have a virtual destructor.)

vs_vector was only ever used in one place (in this file). It can be replaced with std::vector again, I think. Then if we want to enforce bounds checking (which we probably should) we can change all occurrences of `operator[]` with `.at()`.

Reply:
Cool.

Reply:
This is #987, which is waiting on #974.